creatorkit/Toolbox.md

```diff
@@ -0,0 +1,135 @@
+
+<code>$ sudo tar -zxvf ~/downloads/rootfs.tar -C /media/home/name_of_storage_device</code>
+
+<code>$ sync</code>
+
+### Booting from USB Drive/SD Card
+
+In order to boot from USB you need to connect it via an On-the-go cable to the microUSB port next to the power connector. Once that is connected reboot the board. You will then have a window of opportunity to interrupt the default boot process (booting from the flash memory) by pressing a key in the serial terminal. This will give you access to the bootloader terminal.
+
+To boot from USB:
+
+<code># run usbboot</code>
+
+To boot from microSD:
+
+<code># run mmcboot</code>
+
+## Connecting Ci40 to the Internet
+
+Ci40 supports Ethernet and WiFi connections.
+
+To connect to WiFi via SoftAP you need a WiFi capable device (phone,laptop etc.). Power on Ci40 and browse for WiFi networks on your WiFi device. You should find an open WiFi network called "Creator XXXXXXXX". Connect to this network and open a web browser. Navigate to 192.168.1.1 in your browser. You will be presented with a web page that will allow you to choose a network from a list, enter a password and save. Your Ci40 will then connect to this network.
+
+## Programming a 6LoWPAN Clicker
+
+Before programming a Clicker board you need to apply power. You can use batteries, USB or (for purely for programming purposes) a Pickit programmer.
+
+In order to program the 6LoWPAN Clicker you need to connect a Pickit 3.
+
+![serial](pickit.resized.jpg)
+
+Make sure that the arrow indicating Pin1 on the back of the board is matched with the arrow on the Pickit3. Once this is done you need to run the MPLAB X IPE, available [here](http://microchip.wikidot.com/ipe:installation), and choose PIC32MX470F512H as the target device.
+
+If you want to, you can use the Pickit to power your board while you program. To do this, you need to configure MPLAB X IPE to enable the feature. Click "Settings" > "Advanced Mode". Click on the "Power" category on the left and click the checkbox for "Power Target Circuit from Tool".You can then go back to the "Operate" section and proceed.
+
+Select the desired .hex file, and click Program to flash the board.
+
+## Creating a FlowM2M account
+
+In order to use your Creator Kit you will need to create a free account on [the FlowM2M website](http://www.flowcloud.io). This account allows you to manage your connected devices, and also interact with them remotely. You will use a device registration code from this site to provision devices using FlowM2M.
+
+## Provisioning Ci40
+
+Log in to your account on [the FlowM2M website](http://www.flowcloud.io) and view the Dashboard. From here you can navigate to the Devices page. This page will show all previously provisioned devices. In order to add a new device you need to click on the "Register" button, and then take a code.
```

Comment:
Current site is Flowcloud, new site will be FlowM2M. We'll update all the links before trial. Generally we will use flowm2m from now on.